src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -143,18 +144,20 @@ public void edgeOfTheBlockTearing() throws IOException {
                 new HtHead(
                     new InputOf(
                         new JoinedText(
-                            "\r\n",
+                            new TextOf("\r\n"),
                             block,
-                            "",
-                            "text here"
+                            new TextOf(""),
+                            new TextOf("body here")
```

Comment:
@ilyakharlamov in this case I wouldn't have bothered to use `TextOf`, it is a bit too much I think :)

Reply:
@victornoel done in c7a5980